src/js/libgeo/GeoOps.js

```diff
@@ -709,6 +807,22 @@ geoOps.CircleMr.updatePosition = function(el) {
     el.matrix = General.withUsage(matrix, "Circle");
     el.radius = r;
 };
+geoOps.CircleMr.getRandomMove = function(el) {
+    // radius 
+    var r;
+    var oldr = el.radius;
+    var oabs = CSNumber.abs(oldr).value.real;
+
+    // if radius was small we want something larger and if not we scale the old one
+    r = oabs < CSNumber.eps ? CSNumber.real(Math.random() + 0.1) : CSNumber.mult(oldr, CSNumber.real(Math.random() + 0.5));
```

Comment:
I think that a regular `if … else …` might be easier to read here, since the line is pretty long already.
This code chooses only real radii, where most other randomization is complex.
It also performs pretty large changes where most other randomizations tend to do small steps only.
Are there any particular reasons for these?
(Personally I'm all for making large changes, but there were concerns that these would make tracing take too long.)